### DISCUSSION TITLE
I love this library!!

### DISCUSSION THREAD

**Discussion by wvddrss** [orval-labs/orval#337] [General]
Maybe I'm biased as Orval is in my top 3 of favorite beers.
But this library is purely awesome, the greatest discovery of 2022 in TS land.

More automation means more time to watch the magic happen while drinking one.

Keep up the good work, cheers!

**Comment by anymaniax** [comment]
Ahah thanks! It's also in my top 3! This kind of message really helps! I started this library because I wanted something easy to use for my clients but sometimes I am wondering if it really helps other people. So thanks again

**Comment by anymaniax** [comment]
Also thanks to @melloware @CPatchane @yekver. To be honest, lately was difficult to find time and I wasn't really motivated. You really help me a lot to keep doing it and also motivate me!

**Reply by melloware** [reply]
I got introduced to react query and realized I didn't want to hand code all my apis.  I was going to write my own OpenApI generator but luckily I found this library.  We appreciate all your hard work @anymaniax !

**Comment by neemski** [comment]
Late to the party but, hear hear 🍺. @anymaniax thanks for the great work! This library has helped our team so much, especially with hitting the ground running with React Query. Glad to see it picking up steam and popularity too, definitely well-deserved 🍻
